docs/docs/configuration/cameras.md

```diff
@@ -89,7 +89,9 @@ An ONVIF-capable camera that supports relative movement within the field of view
 
 ## ONVIF PTZ camera recommendations
 
-This list of working and non-working PTZ cameras is based on user feedback.
+This list of working and non-working PTZ cameras is based on user feedback. It is thus important that you report your camera working or not working by opening an issue or by creating a pull request. 
```

Comment:
```suggestion
This list of working and non-working PTZ cameras is based on user feedback. If you'd like to report specific quirks or issues with a manufacturer or camera that would be helpful for other users, open a pull request to add to this list.
```

The issues queue on this repo is for feature requests, and I think this wording should not feel as "urgent" to get users to report their specific camera because, as noted earlier, we're not looking for this list to be exhaustive.